scenario.py

```diff
@@ -407,8 +409,8 @@ def split_data(self, is_logging_enabled=True):
         # ... or receive different amounts?
 
         # Check the percentages of samples per partner and control its coherence
-        assert len(self.amounts_per_partner) == self.partners_count
-        assert np.sum(self.amounts_per_partner) == 1
+        assert len(self.amounts_per_partner) == self.partners_count, "Error: in .yml in amounts_per_partner arguments: the amounts_per_partners arguments (a list) has to contain a proportion per each partner - Here there is too many or not enough proportion"
+        assert np.sum(self.amounts_per_partner) == 1, "Error: in .yml in amounts_per_partner arguments: the proportions you provided don't sum up to exactly 1"
```

Comment:
I just had an idea: we should relax this test to `<=1` instead of `==1`. If a user wants to use only 40% of the dataset, that is fine, and maybe sometimes even a good idea.